1_managing_resources/create_ssh_server.py

```diff
@@ -28,6 +28,7 @@
     hostname="203.0.113.23",
     username="example",
     port=22
```

Comment:
Need a comma after `port=22`, and then probably on this one and others below, the last item could lose the comma after, right? Just to clean it up, I know it doesn't matter.